Aquarius/Src/Aquarius/Sound/Devices.cpp

```diff
@@ -12,13 +12,19 @@ namespace Aquarius {
             Initialize();
         }
 
+        uniquePtr<Device> Device::Create()
```

Comment:
So this class is not meant to be a singleton right? Just making sure cause the way this is now you could theoretically have multiple audio devices, unlike before

Reply:
I think there probably should only be one audio device, I will switch it back.